### DISCUSSION TITLE
App as Nuget Package in PrismApplication

### DISCUSSION THREAD

**Discussion by Yerodin84** [PrismLibrary/Prism#2412] [WPF]
Is there a way another app (than NuGet) to show inside the Content Region?
When I open this as a new window it works fine ... but not when I want to add the window control to the regionmanager?

**Comment by brianlagunas** [comment]
You can't show a Window in a region. This is not supported by WPF. You would need to grab the content of the window and add it to the region instead

**Reply by Yerodin84** [reply]
Ok,thanks a lot.
We have now solved the problem differently. For us it's better to solve this as a Module because we can use the complete Style of the Main Program.
